theme/static/js/hs_file_browser.js

```diff
@@ -1669,7 +1673,7 @@ $(document).ready(function () {
         // only need that path after /data/contents/
         var path = file.attr("data-url").split('/data/contents/')[1];
         var fullURL;
-        if (~$(this).attr("url_aggregation").indexOf("HS_JS_AGG_KEY")) {
+        if ($(this).attr("url_aggregation")) {
```

Comment:
Should ```indexOf()``` call be retained so that the replacement only happens when it is needed?

Reply:
it's not needed, the replace will just return the string unchanged if that key doesn't.  This change allows urls that have HS_JS_MAIN_FILE_KEY and not HS_JS_AGG_KEY.